### ISSUE TITLE
About y-axe translations

### ISSUE THREAD

**Issue by shenao1994** [eigenvivek/DiffDRR#224]
Hi! I saw that you updated the code of diffdrr. In the tutorials, I found that **translations=torch.tensor ([[0.0, 850.0, 0.0]], device=device)**. If I change the data, what settings should Y-axis be based on.

Thank you！

**Comment by eigenvivek** [comment]
Hi @shenao1994, I need to make a dedicated notebook going over the new geometry in DiffDRR (this pull request summarizes a lot of the changes - #207), but I can provide a quick example here:

- The isocenter of your CT is now placed at the origin of world coordinates
- The camera (i.e., X-ray source) is placed at the origin and the detector plane at (0, -f, 0) where f is the C-arm's focal length (source-to-detector *distance*, no longer the radius)
- Increasing y moves the camera further away from the patient
- You can mimic this by pointing your finger into your chest (your finger is the camera). Increasing y moves your finger further from your chest.

These changes were made to make the camera in DiffDRR better mimic physical C-arms. In the DICOM of some C-arm images, you'll find spatial parameters encoded like the source to detector distance, source to isocenter distance, primary positioner angle, and secondary positioner angle. DiffDRR now can directly decode these parameters to produce a camera pose.

But as I said, this is still under active development and I need to make a dedicated tutorial notebook for how this geometry works.

Please let me know what further questions you may have!

**Comment by shenao1994** [comment]
> Hi @shenao1994, I need to make a dedicated notebook going over the new geometry in DiffDRR (this pull request summarizes a lot of the changes - #207), but I can provide a quick example here:
> 
> * The isocenter of your CT is now placed at the origin of world coordinates
> * The camera (i.e., X-ray source) is placed at the origin and the detector plane at (0, -f, 0) where f is the C-arm's focal length (source-to-detector _distance_, no longer the radius)
> * Increasing y moves the camera further away from the patient
> * You can mimic this by pointing your finger into your chest (your finger is the camera). Increasing y moves your finger further from your chest.
> 
> These changes were made to make the camera in DiffDRR better mimic physical C-arms. In the DICOM of some C-arm images, you'll find spatial parameters encoded like the source to detector distance, source to isocenter distance, primary positioner angle, and secondary positioner angle. DiffDRR now can directly decode these parameters to produce a camera pose.
> 
> But as I said, this is still under active development and I need to make a dedicated tutorial notebook for how this geometry works.
> 
> Please let me know what further questions you may have!

Thank you very much for your patient explanation. I am currently working on the registration of real X-ray. Therefore, I need to know how the coordinate system of **diffdrr** is established. My current understanding is that CT volume is transformed into the camera coordinate system of **diffdrr**. I don't know if I understand it correctly. I hope to understand the whole coordinate system transformation process, which is very helpful for subsequent application to real scenes. I will wait for you to give some details about the diffdrr geometric transformation. Thank you again!

**Comment by eigenvivek** [comment]
No problem! I'll send you an email when the geometry tutorial is pushed.
